### DISCUSSION TITLE
Convert full name month char (Indonesia) to month name  (date type/month)

### DISCUSSION THREAD

**Discussion by Talioll** [teamalgoritma/community#423] [Q&A] [answered]
Bagaimana cara mengubah nama bulan dalam bahasa Indonesia dan diubah ke month year? Apakah harus mengubah sys locale? 
![q1](https://user-images.githubusercontent.com/112487401/198818998-25b7faaa-a847-4e2c-b867-d96c2eba5924.png)

**Comment by yosiaazarya** [comment, marked as answer]
Untuk melakukan hal tersebut, tampaknya kita perlu unuk mengubah ke bahasa inggris terlebih dahulu baru diubah menjadi tipe data waktu. Contoh yang dapat dilakukan adalah sebagai berikut : https://stackoverflow.com/questions/55828473/translate-months-in-r-from-italian-to-english
